solutions/usaco-989.mdx

```diff
@@ -2,46 +2,108 @@
 id: usaco-989
 source: USACO Bronze 2020 January
 title: Race
-author: Sathvik Chundru
+author: Sathvik Chundru, Kevin Sheng
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_race_bronze_jan20.html)
 
 <LanguageSection>
- 
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <fstream>
+
+using std::cout;
+using std::endl;
+
+int fastest_time(int dist, int max_speed) {
+	int speed_up_dist = 0;  // Amount of distance where Bessie's speeding up
```

Comment:
maybe instead of that comment say that he will speed up at the start (or lhs) but slow down at the end

Reply:
doens't really matter at this point

Reply:
it does because they won't know that slow down dist goes from the end

Reply:
where else could she be slowing down?

Reply:
meh then it's your choice